frontend/src/lib/components/Cards/InsightCard/InsightCard.tsx

```diff
@@ -217,266 +209,178 @@ function InsightMeta({
-    }px + 3.5rem /* margins */ + 3px /* border and spacer */)`
-    const transitionStyles = primaryHeight
-        ? {
-              entering: {
-                  height: unfoldedHeight,
-              },
-              entered: {
-                  height: unfoldedHeight,
-              },
-              exiting: { height: foldedHeight },
-              exited: { height: foldedHeight },
-          }
-        : {}
 
     return (
-        <CSSTransition in={areDetailsShown} timeout={200} classNames="InsightMeta--expansion">
-            {(transitionState) => (
-                <div className="InsightMeta" style={transitionStyles[transitionState]}>
-                    <div className="InsightMeta__primary" ref={primaryRef}>
-                        {ribbonColor &&
-                            ribbonColor !==
-                                InsightColor.White /* White has historically meant no color synonymously to null */ && (
-                                <div className={clsx('InsightMeta__ribbon', ribbonColor)} />
+        <CardMeta
+            setPrimaryHeight={setPrimaryHeight}
+            ribbonColor={ribbonColor}
+            showEditingControls={showEditingControls}
+            showDetailsControls={showDetailsControls}
+            setAreDetailsShown={setAreDetailsShown}
+            areDetailsShown={areDetailsShown}
+            topHeading={
+                <>
+                    <span title={INSIGHT_TYPES_METADATA[filters.insight || InsightType.TRENDS]?.description}>
+                        {INSIGHT_TYPES_METADATA[filters.insight || InsightType.TRENDS]?.name}
+                    </span>{' '}
+                    • {dateFilterToText(filters.date_from, filters.date_to, 'Last 7 days')}
+                </>
+            }
+            meta={
+                <>
+                    <Link to={urls.insightView(short_id)}>
+                        <h4 title={name} data-attr="insight-card-title">
+                            {name || (
+                                <i>
+                                    {summarizeInsightFilters(filters, aggregationLabel, cohortsById, mathDefinitions)}
+                                </i>
                             )}
-                        <div className="InsightMeta__main">
-                            <div className="InsightMeta__top">
-                                <h5>
-                                    <span
-                                        title={
-                                            INSIGHT_TYPES_METADATA[filters.insight || InsightType.TRENDS]?.description
+                        </h4>
+                    </Link>
+
+                    <div className="CardMeta__description">{insight.description || <i>No description</i>}</div>
+                    {insight.tags && insight.tags.length > 0 && <ObjectTags tags={insight.tags} staticOnly />}
+                    <UserActivityIndicator at={insight.last_modified_at} by={insight.last_modified_by} />
```

Comment:
We should be consistent with this. If "Last modified by" is in the foldable section in text cards, it should be in the foldable section here too (next to "Created by")

Reply:
yep, will push this into #12560 wanted to keep this as small as I could get away with